### PR TITLE
validators directory: Add StakeLab

### PR DESCRIPTION
**Pull request by zdeadex**
Key TBD

### CODE REVIEW

**Comment by VT-Cosmos** [comment]
Thank you for submitting your request @zdeadex. We are going to get back to you shortly regarding the next steps.

**Comment by VT-Cosmos** [comment]
@zdeadex 
Thank you for your submission.
There were some updated point that are missing from the application. 
* Unique validator key to submit (from Cosmos Hub)
* Region you and/or your validator is based (country)

Make sure to check the updated requirements in the README to ensure all is covered in the submission, before the deadline.

**Comment by zdeadex** [comment]
GenTX added and  Region too
Is the unique validator key the gentx right? 

cc @VT-Cosmos

**Comment by VT-Cosmos** [comment]
@zdeadex 
We are moving the validator communication to a private group in the GovGen/AtomOne Discord. Which you can find here: https://discord.gg/atomone
Please join using the handle provided in your validator PR, or edit your user profile so that it designates your validator so that we can easily change your role

File: govgen-1/gentx/gentx-148d010cbeb45140f190abfee435f42347d2c195.json [hunk 1]
```diff
@@ -0,0 +1 @@
...escription":{"moniker":"StakeLab","identity":"F12B081334CBE0C6","website":"https://www.stakelab.zone","security_contact":"security@stakelab.zone","details":"Grow your assets - Staking \u0026 Relaying Hub for Cosmos ecosystem"},"commission":{"rate":"0.000000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"govgen1sv24u5ztrw6dfzhazlqmyz2nrytlkx2cd7dnzu","validator_address":"govgenvaloper1sv24u5ztrw6dfzhazlqmyz2nrytlkx2cuuyjgx","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"VRQubC7yPu/JGfHbDAcCYAD17czoPexG2xH02Sww62A="},"value":{"denom":"ugovgen","amount":"1000000"}}],"memo":"148d010cbeb45140f190abfee435f42347d2c195@65.108.71.140:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AyLE2BV++hAyf0W1Nb/KFHmREbeI09Ve/dNPvOk2nKbC"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["6w+NgSGaRnJDNFsyKKWTnBnCfcrqvtC7E6loMZ6zaMtQqK/sUvGKs/Kv/ljB1k1ANlwE/KJnZzNcYqakI4v0mg=="]}
```
Comment (albttx) [inline govgen-1/gentx/gentx-148d010cbeb45140f190abfee435f42347d2c195.json:1]: Please update rate to 0

**Comment by giunatale** [comment]
Please update your gentx to follow the recommended values. please submit a gentx using the command as specified in https://github.com/atomone-hub/validator/tree/main/govgen-1#register-in-the-genesis

Your commision values must all be set to 0.

Please do this ASAP or you might be excluded from genesis

**Pull request merged**